Database.cs

```diff
@@ -0,0 +1,102 @@
+﻿using MongoDB.Driver;
+
+namespace InventoryManagmentMongoDB
+{
+    public class Database
```

Comment:
this class does not follow the Single Responsibility Principle (SRP). 
we need to create separate classes: one for managing database access for products and another for managing inventory items. 
introducing ProductRepository and InventoryItemRepository would be beneficial!